table/server/master/src/main/java/alluxio/master/table/AlluxioCatalog.java

```diff
@@ -238,6 +256,11 @@ private void apply(alluxio.proto.journal.Table.AttachDbEntry entry) {
     mDBs.put(dbName, db);
   }
 
+  private void apply(alluxio.proto.journal.Table.DetachDbEntry entry) {
+    String dbName = entry.getDbName();
+    mDBs.remove(dbName);
```

Comment:
how do ufs'es get unmounted from alluxio fs?

Reply:
we're going to leave them mounted